.github/workflows/tools.yml

```diff
@@ -0,0 +1,42 @@
+# Test tools directories that have their own go.mod files, and
+# otherwise wouldn't get tested along with the main go-algorand build.
+name: Test tools modules
```

Comment:
I'm not sure if testing two tools subdirectories need to build entire algorand on three supported platforms, it looks as overkill